src/rvary_fmt_plug.c

```diff
@@ -46,10 +46,10 @@ john_register_one(&fmt_rvary);
 #endif
 
 static struct fmt_tests rvary_tests[] = {
-    {"$rvary$**5AA70358A9C369E0", "QWERTY1"},
-    {"$rvary$**EB59EE07FC74AE77", "PASSWORD"},
-    {"$rvary$**062314297C496E0E", "AAAAAAAA"},
-    {"$rvary$**0FF48804F759193F", "TESTTEST"},
+    {"$rvary$*5AA70358A9C369E0", "QWERTY1"},
+    {"$rvary$*EB59EE07FC74AE77", "PASSWORD"},
+    {"$rvary$*062314297C496E0E", "AAAAAAAA"},
+    {"$rvary$*0FF48804F759193F", "TESTTEST"},
```

Comment:
Does this also require an update to some external tool, one that produces those hashes?

Reply:
The external tool is a PR I submitted to the ENUM tool. The PR is already up to date now.

Reply:
Great. Why do we have an asterisk in there at all? Let's drop it while we can. As I recall, those asterisks were originally introduced by @kholia in some "non-hash" formats to separate their many custom fields, and there's no reason to clutter a simple hash with those here.

Reply:
I am using the asterisks as they are what the format for the "siblings" formats use. For a reference, take a look at the other RACF formats and racf2john.

I am ready with the reverted version plus small fixes (BO, whitespaces, indentation type/size, no fake empty salt).
We need just to choose the format and I'm ready to push.

Reply:
I took a look at the sibling formats, and I think we should completely drop the asterisks from this one while we can.

Reply:
Dropped then :)